db/migrate/20170517233803_add_figure_to_pieces.rb

```diff
@@ -0,0 +1,6 @@
+class AddFigureToPieces < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pieces, :figure, :string
```

Comment:
"Figure" is pawn, rook, knight, king, queen, or bishop.  I thought we might need number to distinguish between the different ones of each of those, but the more I think about it, the more we probably don't need it.

In our initial schema, we used "Type" to denote what I'm using "Figure" for now.  However, from what I read,  Ruby requires that we use "Type" in Single Table Inheritance to work.  In order for each instance of a piece to inherit from the Piece class, the type of each instance of a piece must be "Piece".  Does that make sense?